Casks/p/pdf-over.rb

```diff
@@ -1,20 +1,21 @@
 cask "pdf-over" do
   arch arm: "aarch64", intel: "x86_64"
-  folder = on_arch_conditional arm: "pdf-over-for-macos-aarch64/", intel: "pdf-over-mac/"
 
-  version "4.4.6.1"
+  version "4.4.6.1,2025,09"
   sha256 arm:   "0e3d3faee6fdabcd1ea2e863eb66dd247f7a7d952497f1bd0c746cf5b78f35ec",
          intel: "6c1f2424ed851343dd9a7c336f84731faac503b14ab6203dd66fa41a1bb027e7"
 
-  url "https://technology.a-sit.at/download/#{folder}?version=#{version.dots_to_hyphens}"
+  url "https://technology.a-sit.at/wp-content/uploads/#{version.csv.second}/#{version.csv.third}/PDF-Over-#{version.csv.first}-#{arch}.dmg"
   name "PDF-Over"
   desc "Digitally sign PDFs with the Austrian Buergerkarte or ID Austria"
   homepage "https://technology.a-sit.at/pdf-over/"
 
   livecheck do
-    url "https://technology.a-sit.at/download/#{folder}"
-    regex(/PDF[._-]Over[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg/i)
-    strategy :header_match
+    url :homepage
+    regex(%r{href=.*?uploads/(\d+)/(\d+)/PDF[._-]Over[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/(\d+)/(\d+)/PDF[._-]Over[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg}i)
```

The .../YYYY/MM/file-name.dmg path format is unique enough that we should be safe omitting the leading `uploads` from the regex. That is to say, the regex produces the same matches without `uploads` in the current content and that doesn't seem likely to change (unless upstream disables the upload month/date organization setting in WordPress).